docs/resources/monitor.md

```diff
@@ -16,44 +16,45 @@ resource "uptimerobot_monitor" "my_website" {
   type          = "http"
   url           = "http://example.com"
 }
+
+resource "uptimerobot_monitor" "my_cron" {
+  friendly_name = "My Cronjob"
+  type          = "heartbeat"
+  interval      = 300
+}
 ```
 
 ## Arguments Reference
 
-* `friendly_name` - friendly name of the monitor (for making it easier to distinguish from others).
-* `url` - the URL/IP of the monitor.
-* `type` - the type of the monitor. Can be one of the following:
+* `api_key` - (Required) uptimerobot API key
+* `friendly_name` - (Required) friendly name of the monitor (for making it easier to distinguish from others).
+* `url` - (Required for all types EXCEPT for the heartbeat one) the URL/IP of the monitor.
+* `type` - (Required) the type of the monitor. Can be one of the following:
   - *`http`*
+  - *`heartbeat`*
   - *`keyword`* - will also enable the following options:
     - `keyword_type` - if the monitor will be flagged as down when the keyword exists or not exists. Can be one of the following:
-      - `exists`
-      - `not exists`
-    - `keyword_value` - the value of the keyword.
+      - `exists` -  (required for keyword monitoring)
+      - `not exists` -  (required for keyword monitoring)  
+    - `keyword_value` -  (required for keyword monitoring) the value of the keyword.
   - *`ping`*
   - *`port`* - will also enable the following options:
-    - `sub_type` - which pre-defined port/service is monitored or if a custom port is monitored. Can be one of the following:
+    - `sub_type` - (Required for port monitoring and custom) which pre-defined port/service is monitored or if a custom port is monitored. Can be one of the following:
       - `http`
       - `https`
       - `ftp`
       - `smtp`
       - `pop3`
       - `imap`
       - `custom`
-    - `port` - the port monitored (only if subtype is `custom`)
-* `http_method` - available for HTTP monitoring. Can be one of the following:
-  - `HEAD` (default for non-keyword)
-  - `GET` (default for keyword)
-  - `POST`
-  - `PUT`
-  - `PATCH`
-  - `DELETE`
-  - `OPTIONS`
```

Comment:
I think this was accidentally swallowed in the refactor :)

Reply:
Yep, you are right, thanks for the review.
Fixed  - 84523438ede1594ed18f7fbc921770315f5ea895